lib/SILOptimizer/SILCombiner/SILCombinerMiscVisitors.cpp

```diff
@@ -1006,6 +1008,39 @@ SILInstruction *SILCombiner::visitCondBranchInst(CondBranchInst *CBI) {
       Default = isFalse ? CBI->getFalseBB() : Default = CBI->getTrueBB();
```

Comment:
Shouldn't be there a check if the default result is an IntegerLiteralInst?
Also, in this line the second assignment in the ?: expression is a typo (doesn't harm but looks weird)